client/layout/global-sidebar/hooks/use-global-sidebar.js

```diff
@@ -0,0 +1,9 @@
+export const useGlobalSidebar = ( siteId, sectionGroup ) => {
+	// TODO: need to expand on logic here - could add feature flag or some environment variable to control this
+	const shouldShowGlobalSidebar =
+		sectionGroup === 'me' ||
+		sectionGroup === 'reader' ||
+		sectionGroup === 'sites-dashboard' ||
+		( sectionGroup === 'sites' && ! siteId );
+	return { shouldShowGlobalSidebar };
+};
```

Comment:
From Eru's earlier PR 

```suggestion
import { isEnabled } from 'calypso/server/config';

export const useGlobalSidebar = ( siteId, sectionGroup ) => {
	let shouldShowGlobalSidebar = false;
	if ( isEnabled( 'layout/dotcom-nav-redesign' ) ) {
		shouldShowGlobalSidebar =
			sectionGroup === 'me' ||
			sectionGroup === 'reader' ||
			sectionGroup === 'sites-dashboard' ||
			( sectionGroup === 'sites' && ! siteId );
	}
	return { shouldShowGlobalSidebar };
};
```

Reply:
Did some smoke testing and I can confirm that adding the flag here does the job of gating all changes in this PR.

Reply:
Committed, thanks for the suggestion 👍